examples/java/README.md

```diff
@@ -0,0 +1,18 @@
+## Run
+
+Ensure that you have an instance of ValKey running on "localhost" on "6379". Otherwise, update glide.examples.ExamplesApp with a configuration that matches your server settings.
```

Comment:
This is the README for examples/java. So I think we need to change `glide.examples.ExampleApp` to both standalone and cluster examples provided within this example/java folder.
I can make this change after your merge for you.

Reply:
Ok.  I'll let you update that in the follow-up PR.  This PR just moves the folder contents.